pageserver/src/tenant/ephemeral_file.rs

```diff
@@ -36,12 +36,15 @@ pub struct EphemeralFiles {
 }
 
 pub struct EphemeralFile {
-    file_id: u64,
+    ephemeral_files_id: u64,
```

Comment:
```suggestion
    ephemeral_file_id: u64,
```
Do we need a separate ephemeral file ID and page cache file ID, or could they be the same?